src/providers/bitbucket.ts

```diff
@@ -0,0 +1,40 @@
+import { OAuthConfig, OAuthUserConfig } from "./oauth"
+
+interface BitbucketProfile {
+  username: string
+  account_id: string
+  display_name: string
+  links: {
+    avatar: {
+      href: string
+    }
+  }
+}
+
+export default function Bitbucket<P extends Record<string, any> = BitbucketProfile>(
+  options: OAuthUserConfig<P>
+): OAuthConfig<P> {
+  return {
+    id: "bitbucket",
+    name: "Bitbucket",
+    type: "oauth",
+    authorization: {
+      url: "https://bitbucket.org/site/oauth2/authorize",
+      params: {
+        scope: ""
+      }
+    },
```

Comment:
```suggestion
    authorization: "https://bitbucket.org/site/oauth2/authorize",
```

This could be simplified, I believe

Reply:
It returns an error when passing without the empty `scope`:
```sh
[next-auth][error][OAUTH_CALLBACK_HANDLER_ERROR]
https://next-auth.js.org/errors#oauth_callback_handler_error invalid_scope {
  error: {
    message: 'invalid_scope',
    stack: 'Error: invalid_scope\n' +
      '    at oAuthCallback (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next-auth/core/lib/oauth/callback.js:37:19)\n' +
      '    at Object.callback (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next-auth/core/routes/callback.js:50:39)\n' +
      '    at NextAuthHandler (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next-auth/core/index.js:103:41)\n' +
      '    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n' +
      '    at async NextAuthNextHandler (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next-auth/next/index.js:40:7)\n' +
      '    at async /home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next-auth/next/index.js:80:32\n' +
      '    at async Object.apiResolver (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next/dist/server/api-utils.js:102:9)\n' +
      '    at async DevServer.handleApiRequest (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next/dist/server/next-server.js:1017:9)\n' +
      '    at async Object.fn (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next/dist/server/next-server.js:904:37)\n' +
      '    at async Router.execute (/home/arthurfalcao/Development/arthurfalcao/epic-notes-generator/node_modules/next/dist/server/router.js:210:32)',
    name: 'Error'
  },
  error_description: 'Unknown scope: openid',
  body: '',
  providerId: 'bitbucket',
  message: 'invalid_scope'
}
```